lib/spack/spack/package.py

```diff
@@ -82,6 +82,16 @@
 _spack_configure_argsfile = 'spack-configure-args.txt'
 
 
+def has_test_method(pkg):
```

Comment:
Minor: can we add a docstring?

Reply:
@tldahlgren We also need to update command completion:
```console
$ spack commands --update-completion
```

Reply:
> @tldahlgren We also need to update command completion:
> 
> ```
> $ spack commands --update-completion
> ```

I was hoping to hear back whether I'd gone too far with the options but it's a quick command so it's not really an issue.

Reply:
> I was hoping to hear back whether I'd gone too far with the options

Initially I thought so, but looking at the fact that we have `--all` I believe it's better to converge to a default output that doesn't force you to scroll up to search basic information. I posted in Slack to also hear other people opinions, but :heavy_check_mark: for me.